sensor_msgs/PointCloud2[] points
---
sensor_msgs/PointCloud2 points
